cmd/sql/flow.go

```diff
@@ -149,6 +143,14 @@ func executeValidate(cmd *cobra.Command, args []string) error {
 	projectDirAbsolute := mountDirs[0]
 	args = []string{projectDirAbsolute}
 
+	if environment != "" {
+		flags["env"] = environment
+	}
+
```

Comment:
We also need to add the connection flag for the Validate command.

Reply:
Oh good catch @pankajkoti !